contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -2509,3 +2510,137 @@ get_owner_of_schema(const char *schema)
 
 	return result;
 }
+
+/* Find namespace oid of a procedure based on proc name. */
+static Oid
+get_proc_namespace_oid(char **proc_name, char *curr_db)
+{
+	char *physical_sch_name;
+	char *db_name;
+	char *schema_name;
+	char	  **splited_object_name;
+	Oid obj_schema_oid = InvalidOid;
+
+	/* Resolve the three part name. */
+	splited_object_name = split_object_name(*proc_name);
+	db_name = splited_object_name[1];
+	schema_name = splited_object_name[2];
+	*proc_name = splited_object_name[3];
```

Comment:
Can any of them be NULL?

Reply:
None of these could be NULL as per the definition`split_object_name` we fill unspecified parts with empty strings.
I have added a check that the incoming `proc_name` should not be null and then only we will call splited_object_name on it.